README.md

```diff
@@ -144,6 +144,11 @@ This is the main Robot-Centric Elevation Mapping node. It uses the distance sens
 
         rosservice call /elevation_mapping/clear_map
 
+* **`save_map`** ([grid_map_msgs/ProcessFile])
+
+    Saves the current fused grid map and raw grid map to a rosbag file. Field `topic_name` must be a base name, i.e. no leading slash character (/). If field `topic_name` is empty, then `elevation_map` is used per default.
+
+        rosservice call /elevation_mapping/save_map
```

Comment:
Parameters needed? Please test.

Reply:
you're right, I forgot to put the parameters. Now it should be good to go ;)